lua/wikis/valorant/MatchGroup/Input/Custom.lua

```diff
@@ -116,6 +117,16 @@ function MatchFunctions.getExtraData(match, games, opponents)
 	}
 end
 
+---@param match table
+---@param games table[]
+---@return string?
+function MatchFunctions.getPatch(match, games)
+	return Logic.emptyOr(
+		#games > 0 and games[1].patch or nil,
+		match.patch
```

Comment:
Wouldn't the other order make more sense? Or at least check that all games have the same (or none-at-all) patch

Reply:
I know in dota2 have matches with different patches in games.

But taking the manual input of match patch seems to be a good thing, allowing editors to override incorrect data